src/main/java/com/uid2/admin/vertx/service/SiteService.java

```diff
@@ -388,13 +377,33 @@ private void handleSiteUpdate(RoutingContext rc) {
                     ResponseUtil.error(rc, 400, "Invalid parameter for visible: " + visibleParam);
                 }
             }
+            if (name != null) {
+                if (validateSiteName(rc, name)) return;
```

Comment:
It appears the convention in this file (which I also prefer) is that if statements have a following brace: 
if (validateSitenName(rc, name)) {
    return;
}